data-raw/process_data_sets.R

```diff
@@ -381,3 +381,12 @@ avocados <-
   mutate(type = as.factor(type)) %>%
   rename(xlarge_hass_sold = xlarage_hass_sold)
 usethis::use_data(avocados, overwrite = TRUE)
+
+
+# saratoga housing prices
+# original google sheet here: https://docs.google.com/spreadsheets/d/1AY5eECqNIggKpYF3kYzJQBIuuOdkiclFhbjAmY3Yc8E
+# 
+saratoga_houses <- "data-raw/saratoga_houses.csv" %>%
+  read.csv("https://wjhopper.github.io/SDS-201/data/saratoga_ny_home_prices.csv")
```

Comment:
This doesn't work. You should pipe the URL into `read_csv()`, which shouldn't be called with any additional arguments. 

Please also run `clean_names()` on the data.frame and make sure to update the names in the documentation so that they are all in snake_case.